src/JsonLD.Entities/Converters/JsonLdLiteralConverter.cs

```diff
@@ -28,7 +29,28 @@ static JsonLdLiteralConverter()
         /// </summary>
         public override void WriteJson(JsonWriter writer, [AllowNull] object value, JsonSerializer serializer)
         {
-            writer.WriteValue(value);
+            var type = value?.GetType();
+            if (type?.HasImplicitlyTypedJsonType() == true)
+            {
+                writer.WriteValue(value);
+                return;
+            }
+
+            writer.WriteStartObject();
+            writer.WritePropertyName(JsonLdKeywords.Value);
+
+            if (value is TimeSpan)
+            {
+                writer.WriteValue(XmlConvert.ToString((TimeSpan)value));
+            }
+            else
+            {
+                var valueString = JsonConvert.ToString(value);
+                writer.WriteValue(valueString.Trim('"')); // for the weirdest of reasons, DateTime is double-quoted
```

Comment:
Uh, it's just me being foolish. The date is in quotes because that is a date is represented in JSON as string. Numbers aren't so they aren't in quotes. Same with bool